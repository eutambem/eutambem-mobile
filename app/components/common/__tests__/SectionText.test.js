import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';

import SectionText from '../SectionText';

const props = {
  title: 'Test SectionText',
};

describe('SectionText', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<SectionText {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
