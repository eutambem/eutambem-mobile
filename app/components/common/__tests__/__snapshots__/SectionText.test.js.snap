// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionText renders correctly 1`] = `
<ForwardRef(View)
  style={
    Object {
      "marginTop": 10,
    }
  }
>
  <ForwardRef(Text)
    style={
      Object {
        "color": "#000000",
        "fontSize": 14,
      }
    }
  >
    Test SectionText
  </ForwardRef(Text)>
</ForwardRef(View)>
`;
